Documentation/Configuration/Yaml/ServicesYaml.rst

```diff
@@ -24,5 +24,8 @@ A typical :file:`Configuration/Services.yaml` may look like this:
 
 .. seealso::
 
-   * TYPO3 uses the Symfony Dependency Injection component, so official documentation can be found at
+   * TYPO3 uses the Symfony Dependency Injection component. The official documentation can be found at
      https://symfony.com/doc/current/service_container.html
+
+   * The technology used for inluding files and folders is called
```

Comment:
This sentence is not really correct. The pattern with the star is called glob pattern. An explanation is also linked form the Symfony docs for the service container.

However, it is just the pattern, not "technology for including files". It is a very common pattern which we do not have to explain really. But if we do then I would prefer a source without  advertisement like the one that symfony is citing: https://en.wikipedia.org/wiki/Glob_(programming)

Reply:
A link to Wikipedia makes no sense, because everybody knows this lexicon and it also does not explain the matter so good.